project.clj

```diff
@@ -1,25 +1,22 @@
 (def VERSION "0.9.0-SNAPSHOT")
 
-(def VERSION-FORM `(do (require 'cider-nrepl.plugin)
```

Comment:
We read in `plugin.clj`. It's a somewhat lame setup and I've been meaning to change this for a while.